docs/docs/gatsby-cli.md

```diff
@@ -26,10 +26,46 @@ You can also use the `package.json` script variant of these commands, typically
 
 ### `new`
 
-`gatsby new gatsby-site`
+```shell
+gatsby new [<site-name> [<starter-url>]]
```

Comment:
What's the intent of the nested square brackets?

Reply:
that's the common syntax definition that these are optional 


Nested: you can leave off the starter URL or starter-name and starter URL. But you can leave off the site name and out give the starter-u


http://docopt.org/

Reply:
Cool, thanks for the info! That works.